package.json

```diff
@@ -19,23 +19,25 @@
   "version": "0.10.0-0",
   "description": "Flux/CQRS patterns for JavaScript app.",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "directories": {
     "test": "test"
   },
   "scripts": {
     "start:docs": "gitbook serve",
     "build:docs": "gitbook build",
-    "build": "npm-run-all -p build:tsc build:cp_obj -s build:down_transform",
+    "build": "npm-run-all -s clean build:src build:down_transform",
     "build:cp_obj": "cpx 'src/**/*.{js, jsx}' __obj/ --preserve",
     "build:tsc": "tsc --project .",
-    "build:down_transform": "NODE_ENV=production babel __obj --out-dir lib --source-maps",
-    "watch": "babel src --out-dir lib --watch --source-maps",
+    "build:src": "npm-run-all -p build:tsc build:cp_obj",
+    "build:down_transform": "cross-env NODE_ENV=production babel __obj --out-dir lib --source-maps",
```

Comment:
I feel we should sort script names by the order of breaking down a problem.  So `build` -> `build:src` -> (`build:tsc` and `build:cp_obj`) -> `build:down_transform`